src/controller/articlesController.js

```diff
@@ -216,11 +240,43 @@ class ArticleController {
   * @memberof ArticleController
  */
   static async bookmarkArticle(req, res) {
-    const name = req.body.name || req.article.dataValues.title;
     const userId = req.decoded.id;
     const { articleId } = req.params;
 
     try {
+      const userInToken = await findDatabaseField.UserInToken(userId);
+
+      if (!userInToken) {
+        return res.status(404).json({ message: 'User does not exist' });
+      }
+
+      const isValidId = await uuidV4Validator(articleId);
+
+      if (!isValidId) {
+        return res.status(400).json({
+          success: false,
+          message: 'Invalid Id'
+        });
```

Comment:
I feel it's unnecessary to have a descriptive message in this instance as the user hitting the endpoint knows the id that's invalid - It's always the id in the params. Using a descriptive message in this instance would lead to complications.